server/src/controllers/Auth/middleware.ts

```diff
@@ -130,3 +130,19 @@ export function handleAuthenticationError(
     });
   }
 }
+
+export function handleError(
+  err: any,
+  _req: Request,
+  res: Response,
+  next: NextFunction,
+) {
+  if (res.headersSent) {
+    return next(err);
+  }
+  if (err) {
+    return res.status(401).send({
```

Comment:
```suggestion
    return res.status(500).send({
```
It's a server error after all.